app/graphql/types/hmis_schema/organization.rb

```diff
@@ -0,0 +1,26 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Organization < Types::BaseObject
+    include Types::HmisSchema::HasProjects
+
+    description 'HUD Organization'
+    field :id, ID, null: false
+    field :name, String, null: false
+    projects_field :projects, 'Get a list of projects for this organization'
+
+    def projects(**args)
+      resolve_projects(object.projects, **args)
+    end
+
+    def self.organizations(scope = GrdaWarehouse::Hud::Organization.all, user:)
+      scope.viewable_by(user)
```

Comment:
Same here

Reply:
That `def self.organizations` method isn't actually being used, it should be replaced by the `resolve_organizations` method in `HasOrganizations` above, but I must have forgotten to remove it here